### PR TITLE
[15.0][MIG] website_sale_resource_booking: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT41031

@chienandalu @CarlosRoca13 please review!

### CODE REVIEW

File: website_sale_resource_booking/tests/test_ui.py [hunk 1]
```diff
@@ -40,9 +51,9 @@ def test_checkout(self):
         self.start_tour(
             "/shop?search=test not bookable product",
             "website_sale_resource_booking_checkout",
-        )
-        # Find Mr. A's cart
-        so = self.env["sale.order"].search([("partner_id", "=", "Mr. A")])
+            login="ptl",
```
Comment (chienandalu) [inline website_sale_resource_booking/tests/test_ui.py]: Why changing the test?

Comment (stefan-tecnativa) [reply website_sale_resource_booking/tests/test_ui.py]: The test was in conflict with the website_require_login module. To solve that, a user is created and logged in. The behavior is the same as if it were being performed without being logged in.

**Review by CarlosRoca13 — APPROVED** [review]
Code review :+1:

**Comment by pedrobaeza** [comment]
Please rebase and remove dependencies, @stefan-tecnativa

File: website_sale_resource_booking/tests/test_ui.py (hunk 1 above)
Comment (chienandalu) [reply website_sale_resource_booking/tests/test_ui.py]: Ok, anyway give it a more comprehensive or less generic name and login or you could clash again against other module's data

**Review by chienandalu — APPROVED** [review]
Aside from functional test and code review which looks ok, a final tiny comment:

**Comment by stefan-tecnativa** [comment]
> Aside from functional test and code review which looks ok, a final tiny comment:

Change done. It's true, a more appropiate name is a better option.

**Review by chienandalu — APPROVED** [review]

**Review by NICO-SOLUTIONS — COMMENTED** [review]
Functional review. looks good

**Comment by chienandalu** [comment]
One of the CI jobs is failing. Maybe a random fail?

**Comment by stefan-tecnativa** [comment]
> One of the CI jobs is failing. Maybe a random fail?

I will check it.

**Comment by CarlosRoca13** [comment]
Please squash last commit with migration one

**Review by chienandalu — APPROVED** [review]

**Pull request merged**